pkgs/by-name/xc/xcursor-pro/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  stdenvNoCC,
+  lib,
+  fetchurl,
+  variants ? [
+    "Dark"
+    "Light"
+    "Red"
+  ],
+}:
+stdenvNoCC.mkDerivation rec {
```

Comment:
I know this is standard practice. But personally, I'd use a let...in expression.